src/pages/Admin/AdminUpdateClass/AdminUpdateClass.tsx

```diff
@@ -28,13 +64,23 @@ const AdminUpdateClass = () => {
         <AddClassButton type="button" onClick={toggle}>
           + New Class
         </AddClassButton>
-        <Modal titleText="Add New Class" isOpen={isOpen} hide={toggle}>
-          Hello, This is where you will add New Class content
+        <Modal
+          titleText="Add New Class"
+          isOpen={isOpen}
+          hide={toggle}
+          primaryAction={
+            <PrimaryButton type="submit" form="addClassForm">
+              Submit
+            </PrimaryButton>
+          }
+          secondaryAction={<Button onClick={handleCloseModal}>Cancel</Button>}
```

Comment:
```suggestion
          secondaryAction={<Button onClick={toggle}>Cancel</Button>}
```
this should be `handleCloseModal` if you have the reset modal form values in the close handler, but I don't think you need to fetch classes when you hit the cancel button, since there's no update or anything made.

Reply:
hmmm... Let's discuss this, for now the modal doesn't close on submission to allow users to add another class.  For now it only shows user a success message that "Hey, the class has been successfully added"   To close the modal the user would have to click cancel, hence I added a fetch call there.